.buildkite/pipeline.yml

```diff
@@ -77,7 +77,7 @@ steps:
           context: "UI Tests (iPhone)"
 
   - label: "🔬 UI Tests (iPad)"
-    command: .buildkite/commands/run-ui-tests.sh UITests "iPad Air (4th generation)" 15.0
+    command: .buildkite/commands/run-ui-tests.sh iPadOS "iPad Air (4th generation)" 15.0
```

Comment:
I decided to use iOS and iPadOS instead of iPhone and iPad because we can technically run the tests on an iOS destination that is not an iPhone, i.e. the iPod Touch Simulator.

I'm not sure whether this ends up being confusing, though, and I ought to stick to iPhone and iPad. After all, I don't see us testing on iPod Touch any time soon.

What do you think?